F21/yzhang120/lower_case.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+
```

Comment:
Adding comments to your functions to give an overview of what they do would help with readability/understandability